### PR TITLE
New package: GoldenCheetah-3.5

### PR DESCRIPTION
**Pull request by gt7-void**
Performance Software for Cyclists, Runners and Triathletes

http://www.goldencheetah.org/

Free alternative to garmin connect.

NOTE: qwt is included in the source, it is a /different/ branch of qwt than the stable one void ships. I did try to compile using the qwt from void but it fails.

### CODE REVIEW

File: srcpkgs/GoldenCheetah/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'GoldenCheetah'
+pkgname=GoldenCheetah
+version=3.5
+revision=1
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-host-tools bison flex"
+makedepends="qt5-devel qt5-charts-devel qt5-connectivity-devel
+	qt5-multimedia-devel qt5-serialport-devel qt5-svg-devel
+	qt5-webkit-devel zlib-devel"
```
Comment (ericonr) [inline srcpkgs/GoldenCheetah/template:9]: These tabs should be spaces.

**Comment by ericonr** [comment]
How did it fail when using the Void qwt?

File: srcpkgs/GoldenCheetah/template (hunk 1 above)
Comment (gt7-void) [reply srcpkgs/GoldenCheetah/template]: fixed

**Comment by gt7-void** [comment]
> How did it fail when using the Void qwt?

```
In file included from Charts/AllPlot.cpp:20:
Charts/AllPlot.h:26:10: fatal error: qwt_axis_id.h: No such file or directory
   26 | #include <qwt_axis_id.h>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
```
If I remove this `#include` I get stuff like
```
Charts/AllPlot.h:66:7: note: forward declaration of 'class QwtAxisId'
   66 | class QwtAxisId;
```
so it seems to be really necessary.

If you want to have a look, I've pushed gt7-void:GoldenCheetah-qwt with my attempt. Maybe I'm overlooking something.

Thanks for your comments.

**Pull request closed without merging**